config/env/default.js

```diff
@@ -26,6 +26,17 @@ module.exports = {
   // sessionKey is set to the generic sessionId key used by PHP applications
   // for obsecurity reasons
   sessionKey: 'sessionId',
+  // Lusca CSRF config
+  csrf: {
+    csrf: false,
```

Comment:
Not sure about the client side token, however, I like csrf default to false, that way it's a non breaking change and you'd have to enable it and configure it if you wanted it. 

@yilenpan Meet-up with me after-hours sometime and we can investigate the client side token further.